BitTornado/Application/makemetafile.py

```diff
@@ -54,6 +54,34 @@ def make_meta_file(loc, url, params=None, flag=None,
 
     tree = BTTree(loc, [])
 
+    def valid_file(file_obj):
```

Comment:
Nesting functions in the middle of other functions isn't good practise - they could be moved out, and passed whatever parameters they need.

Reply:
I moved these out, but the reason I nested them was to keep params in scope of the make function.